ansible/roles/real-postfix/templates/main.cf.j2

```diff
@@ -0,0 +1,56 @@
+#biff = no
+
+# appending .domain is the MUA's job.
+append_dot_mydomain = no
+
+# Uncomment the next line to generate "delayed mail" warnings
+#delay_warning_time = 4h
+
+alias_maps = hash:/etc/aliases
+alias_database = hash:/etc/aliases
+myorigin = scripts.mit.edu
+mydestination = scripts.mit.edu, scripts, $myhostname, scripts-test.mit.edu, scripts-test, localhost
+relayhost =
+mynetworks_style = host
+mailbox_command_maps =
+    texthash:/etc/postfix/mailbox_command_maps,
+    ldap:/etc/postfix/mailbox-command-maps-ldap.cf
+mailbox_size_limit = 0
+message_size_limit = 41943040
+recipient_delimiter = +
+inet_interfaces = $myhostname{% for vip in vips %}{% if vip.mail | default(False) and (vip.codename|default(ansible_lsb.codename) == ansible_lsb.codename) %}, {{ vip.host }}{% endif %}{% endfor %}
+
+readme_directory = /usr/share/doc/postfix/README_FILES
+sample_directory = /usr/share/doc/postfix/samples
+sendmail_path = /usr/sbin/sendmail
+html_directory = no
+setgid_group = postdrop
+command_directory = /usr/sbin
+manpage_directory = /usr/share/man
+daemon_directory = /usr/libexec/postfix
+newaliases_path = /usr/bin/newaliases
+mailq_path = /usr/bin/mailq
+queue_directory = /var/spool/postfix
+mail_owner = postfix
+# virtual-alias-maps-ldap-reserved.cf redirects {abuse,webmaster,etc.}@ -> root
+# For domains on this pool, virtual-alias-maps-ldap.cf does foo@bar.com -> user+foo@localhost
+# For domains on another pool, virtual-alias-maps-relay-ldap.cf does foo@bar.com -> foo!bar.com@[18.4.86.22]
+# For users on another pool, pass-scripts.mit.edu matches *@scripts.mit.edu and virtual-alias-maps-relay-user-ldap.cf does foo@scripts.mit.edu -> foo!scripts.mit.edu@[18.4.86.22]
+# pass-scripts.mit.edu-suffix and virtual-alias-maps-relay-user-suffix-ldap.cf do the same for foo+bar@scripts.mit.edu
+# Native routing does foo@scripts.mit.edu -> foo
+# smtp_generic_map rewrites foo!scripts.mit.edu@[18.4.86.22] -> foo@scripts.mit.edu after the transport is selected
+# To override, put something in /etc/postfix/force_pool
```

Comment:
force_pool is referenced in virtual_alias_domains, but AIUI virtual_alias_domains is basically just used for set membership checking -- I don't *think* the right hand side matters? Should it be in virtual_alias_maps instead?